corehq/apps/hqadmin/utils.py

```diff
@@ -87,6 +87,17 @@ def get_task_html(detailed_stats, worker_name):
     return mark_safe(worker_status)
 
 
+def parse_celery_pings(worker_responses):
+    pings = {}
+    for worker in worker_responses:
+        assert len(worker.keys()) == 1 and '@' in worker.keys()[0]
+
+        worker_fullname = worker.keys()[0]
+        worker_hostname = worker_fullname.split('@')[1]
+        pings[worker_hostname] = worker[worker_fullname]['ok'] == 'pong'
```

Comment:
we might want to use `worker[worker_fullname].get('ok')` just to be safe so we never fail hard